assets/browser-fetch.js.t

```diff
@@ -38,7 +45,7 @@
       self['default'] = function() {
         pending++;
 
-        return self.fetch.apply(self, arguments).then(function(response){
+        return self.fetch.apply(preferNative ? global : self, arguments).then(function(response){
```

Comment:
I believe this is pulling from the wrong config, see https://github.com/ember-cli/ember-fetch/pull/63/files#r190637528 for more details.